rosetta-source/src/main/resources/result-json-files/fpml-5-13/products/fx-derivatives/fx-ex02-spot-cross-w-side-rates.json

```diff
@@ -101,48 +101,27 @@
                       "meta" : {
                         "globalKey" : "b14144c1"
                       }
-                    },
-                    "exchangeRate" : {
-                      "crossRate" : [ {
```

Comment:
Now that we're deprecating `ExchangeRate` and `CrossRate`, we need a home for these attributes. This is when an exchange rate can be expressed as a product of 2 rates, i.e. a "cross": eg GBP/EUR = GBP/USD x USD/EUR.

`PriceComposite` is the right object for this, but it's incomplete because it doesn't have any unit for the components.